src/scheduler/Scheduler.cpp

```diff
@@ -134,61 +128,23 @@ void Scheduler::vacateSlot()
     thisHostResources.set_usedslots(thisHostResources.usedslots() - 1);
 }
 
-void Scheduler::notifyExecutorFinished(Executor* exec,
```

Comment:
This function was basically just doing the logic of switching executors between the two lists, so can go.